File: example-databinding/src/main/java/com/xwray/groupie/example/databinding/item/DraggableItem.java
package com.xwray.groupie.example.databinding.item;

import androidx.annotation.ColorInt;
import androidx.recyclerview.widget.ItemTouchHelper;

public class DraggableItem extends CardItem {
    public DraggableItem(@ColorInt int colorRes) {
        super();
    }

    @Override public int getDragDirs() {
        return ItemTouchHelper.DOWN | ItemTouchHelper.UP;
    }
}

File: example/src/main/java/com/xwray/groupie/example/item/DraggableItem.kt
package com.xwray.groupie.example.item

import androidx.annotation.ColorInt
import androidx.recyclerview.widget.ItemTouchHelper

class DraggableItem(@ColorInt colorInt: Int) : CardItem(colorInt) {

    override fun getDragDirs(): Int {
        return ItemTouchHelper.UP or ItemTouchHelper.DOWN
    }
}
